jsr352/core/src/main/resources/META-INF/batch-jobs/hibernate-search-mass-indexing.xml

```diff
@@ -39,7 +39,7 @@
         <listeners>
             <listener ref="org.hibernate.search.jsr352.massindexing.impl.steps.lucene.StepProgressSetupListener">
                 <properties>
-                    <property name="isJavaSE" value="#{jobParameters['isJavaSE']}" />
+                    <property name="isJavaSE" value="#{jobParameters['isJavaSE']}?:false;" />
```

Comment:
Actually this parameter is obsolete, I should have removed it. Could you remove it, please?